c_emitter.h

```diff
@@ -145,6 +149,18 @@ class CEmitter
               << "";
     }
 
+    void trusted_function_names()
+    {
+        out() << "const oe_ecall_info_t __" + edl_->name_ +
```

Comment:
This could be made `static` since no other file needs the definition.

Reply:
Done.